lib/MTT/Util.pm

```diff
@@ -490,4 +492,13 @@ sub term_handler{
 	Verbose("All is done, exiting.\n");
 	exit(0);
 }
+
+#--------------------------------------------------------------------------
+
+# Trim the leading and trailing whitespaces
+sub trim_spaces {
+    my ($str) = @_;
+    $str =~ s/^\s*|\s*$//g;
```

Comment:
Fair enough.  But don't forget that that's 4-5 (or more!) orders of magnitude less than the length of a typical MTT client run.  Optimizations are always good, but we probably don't need to optimize for milisecond gains deep down inside MTT.  It's Perl, after all.  :smile: